dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -448,74 +454,220 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
 
         boolean defaultContentToDefaultLangOriginalValue =
                 Config.getBooleanProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", false);
 
-        try {
+        boolean defaultPagetoDefaultLangOriginalValue =
+                Config.getBooleanProperty("DEFAULT_PAGE_TO_DEFAULT_LANGUAGE", true);
 
-            Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", true);
+        try {
+            Config.setProperty("DEFAULT_CONTENT_TO_DEFAULT_LANGUAGE", testCase.defaultContentToDefaultLanguage);
+            Config.setProperty("DEFAULT_PAGE_TO_DEFAULT_LANGUAGE", testCase.defaultPageToDefaultLanguage);
 
             final Host site = new SiteDataGen().nextPersisted();
-            Language lang1 = APILocator.getLanguageAPI().getDefaultLanguage();
-            Language lang2 = new LanguageDataGen().nextPersisted();
+            Language defaultLang = APILocator.getLanguageAPI().getDefaultLanguage();
+            Language secondLang = new LanguageDataGen().nextPersisted();
+            Language thirdLang = new LanguageDataGen().nextPersisted();
             folder = new FolderDataGen().site(site).nextPersisted();
 
-            Contentlet contentletLang1 = TestDataUtils
-                    .getEmployeeContent(true, lang1.getId(), null, site);
-
-            contentletLang1 = contentletAPI.find(contentletLang1.getInode(), systemUser, false);
-            contentletLang1.setStringProperty("firstName", "catherine");
-            contentletLang1 = contentletAPI.checkin(contentletLang1, systemUser, false);
-
-            ContentletDataGen.publish(contentletLang1);
-
-            Contentlet contentletLang2 = contentletAPI
-                    .find(contentletLang1.getInode(), systemUser, false);
-            contentletLang2.setInode("");
-            contentletLang2.setLanguageId(lang2.getId());
-            contentletLang2.setStringProperty("firstName", "catalina");
-            contentletLang2 = contentletAPI.checkin(contentletLang2, systemUser, false);
-
-            ContentletDataGen.publish(contentletLang2);
-
-            final Container container = new ContainerDataGen().withContentType(contentletLang1
-                    .getContentType(), "$!{firstName}").nextPersisted();
-
-            ContainerDataGen.publish(container);
+            Contentlet contentletDefaultLang = null;
+            Contentlet contentSecondLang = null;
 
-            final String uuid = UUIDGenerator.generateUuid();
+            if(testCase.createContentInDefaultLanguage) {
+                contentletDefaultLang = createAndPublishEmployeeContent(site, defaultLang, "catherine");
 
-            final Template template = new TemplateDataGen()
-                    .withContainer(container.getIdentifier(), uuid)
-                    .nextPersisted();
+                if(testCase.createContentInSecondLanguage) {
+                    createNewVersionAndPublishExistingEmployeeContent(secondLang, contentletDefaultLang,
+                            "catalina");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "catalina" appears 8 times in this file; the first occurrence is on line 613](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)